core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -11,6 +12,30 @@ class ReligionManager {
 
     var pantheonBelief: String? = null
     
+    var greatProphetsEarned = 0
+
+    fun clone(): ReligionManager {
+        val clone = ReligionManager()
+        clone.pantheonBelief = pantheonBelief
+        clone.storedFaith = storedFaith
+        return clone
+    }
+
+    fun startTurn() {
+        if (canGenerateProphet() && storedFaith > faithForNextGreatProphet()) {
+            val prophetSpawnChange = (5f + storedFaith - faithForNextGreatProphet()) / 100f
+            if (Random.nextFloat() < prophetSpawnChange) {
+                civInfo.addUnit("Great Prophet")
+                storedFaith = 0 
+                // No, we don't just subtract the amount of faith it requires, you lose ALL of it. Why would we be nice?
```

Comment:
Is this REALLY how the base game works? That's disgusting. If so, I vote we change this.

Reply:
Sadly, this is the case (at least according to the source I found online). I completely agree on changing it though

Reply:
> Is this REALLY how the base game works? That's disgusting. If so, I vote we change this.

so you say the game is open to change disliked things in civ 5 🧐

Reply:
Things that *I* dislike.

Reply:
That's the prerogative of the creator.

Reply:
One could also argue that 'losing faith' isn't necessarily disgusting, it can be a _very healthy_ thing, depending on which cult you ascribe to... I'd vote for moddable, but - your call 😇